perllib/FixMyStreet/App/Controller/Auth.pm

```diff
@@ -87,8 +86,7 @@ sub sign_in : Private {
         }
 
         # unless user asked to be remembered limit the session to browser
-        $c->set_session_cookie_expire(0)
-          unless $remember_me;
+        $c->set_session_cookie_expire(0);
```

Comment:
Is there a reason we're dumping the remember me bit?

Reply:
Given many people don't turn off computers but just sleep (or never turn off mobiles), it's plausible not clicking 'keep me logged in' (so setting a session cookie) would last longer than clicking it.